index.html

```diff
@@ -14,19 +14,24 @@
   <body>
       <!--toolbar section-->
       <main id="header-section">
-        <img src="images/vector.png.png"
-          alt="head-bg"
-          id="headeline-bg"
-        />
-        <header id="page-head">
+      
+        <div id="desk-img">
+          <img src="images/deskimg.jpg" alt="deck" id="desktop-img">
+        </div>
+    
     
           <nav id="navbar">
               <a href="#" id="logo">Welcome</a>
             <a href="#">
-              <i class="fa fa-bars" aria-hidden="true"></i>
+              <i id="menu-bar" class="fa fa-bars" aria-hidden="true"></i>
             </a>
+            <ul class="menu-3">
+              <li> <a href="#">Portfolio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">contact</a></li>
+            </ul>
           </nav>
-        </header>
```

Comment:
The social media icons don't follow the Figma design. Kindly fix it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/173056279-c8288ab6-8884-49a3-a98e-744e131a3c67.png) | ![image](https://user-images.githubusercontent.com/33268927/173056207-d9fd3e04-6f4d-4770-9e4d-d5b4dac57edf.png) |

Reply:
- [ ] As the previous reviewer mentioned, consider to make your social media icons match the design like the Figma design. You can add `padding: 0 29%;` to class `.social-media` to match it.